Mathlib/LinearAlgebra/Dimension/StrongRankCondition.lean

```diff
@@ -371,6 +371,11 @@ theorem rank_span_set {s : Set M} (hs : LinearIndepOn R id s) : Module.rank R 
   rw [← @setOf_mem_eq _ s, ← Subtype.range_coe_subtype]
   exact rank_span hs
 
+theorem toENat_rank_span_set {ι : Type*} {v : ι → M} {s : Set ι}
+    (hs : LinearIndepOn R v s) : (Module.rank R ↑(span R (v '' s))).toENat = s.encard := by
```

Comment:
This is just a bunch of nitpicks but how about:
```suggestion
theorem toENat_rank_span_set {v : ι → M} {s : Set ι} (hs : LinearIndepOn R v s) :
    (Module.rank R <| span R (v '' s)).toENat = s.encard := by
```
I.e.:

- Martin's point about `ι` already existing
- Dropping `↑` (it's not officially in the style guide but I think we should only include these when necessary)
- Using `<|` to avoid `)))`
- When a line break is required, prioritise giving the content after the colon a line to itself